templates/glossary.md

```diff
@@ -0,0 +1,331 @@
+
+[This Zulip discussion](https://leanprover.zulipchat.com/#narrow/stream/113488-general/topic/refl.20taking.2020.20seconds) has a particularly slow example.
+
+### hierarchy
+
+A collection of successively more constrained [typeclasses](#typeclass) within a related area of mathematics.
+In [mathlib](#mathlib), we have the *algebraic hierarchy* (`semiring`, `ring`, `field`, ...), the *order hierarchy* (`preorder`, `partial_order`, `linear_order`, ...), the *topological hierarchy* (`t1_space`, `t2_space`, `normal_space`, ...), the *categorical hierarchy* (`preadditive`, `abelian`, `monoidal`, ...) but also the *scalar hierarchy* (`mul_action`, `distrib_mul_action`, `module`, ...), the *norm hierarchy*, and intersections of previous ones like the *order-algebraic hierarchy*, the *topologico-algebraic hierarchy*, and others.
+
+### `Icc`, `Ico`, `Ioc`, `Ioo`, `Ici`, `Ioi`, `Iic`, `Iio`
+
+Shorthand notation used within [mathlib](#mathlib) for referring to one of 8 kinds of mathematical intervals.
+There are `8` types of intervals, depending on whether the interval is *c*losed, *o*pen or runs to *i*nfinity at each end.
+Names have been made compact by calling each interval `I` + how it ends on the left + how it ends on the right.
+`Iii`, which in the naming convention would refer to an interval infinite at both ends, is not in use.
+
+### infoview
+
+Within the context of interactively editing Lean files, a window or interface which displays incremental [goal](#goal) state, diagnostics, errors and Lean [widgets](#widget) output.
+
+### instance
+
+One of two closely related concepts:
+
+* A [class](#class) argument taken by a `def`, `lemma` or other [declaration](#declaration) which is enclosed by square brackets (`[]`) such that it is resolved by the [typeclass inference](#typeclass-inference) system when the declaration is used.
+* A [declaration](#declaration) created with the eponymous `instance` command, which registers an object with the typeclass inference system for use in the above. As a concrete example, [mathlib](#mathlib) defines an instance of `linear_order` for `ℝ`, enabling reals to be compared with `<`.
+
+
+### Lean Together
+
+An [annual meeting](https://leanprover-community.github.io/lt2021/) for the Lean user community, its [mathlib](#mathlib) library users, as well as the broader theorem prover community.
+
+Previous talks or presentations from the events have been [shared on the community YouTube channel](https://www.youtube.com/channel/UCWe5B7Ikr0AI9727doEUxPg).
+### `leanproject`
+
+A higher-level supporting tool for working with projects, particularly [mathlib](#mathlib), in Lean 3.
+It lives within the [community `mathlib-tools` repository](https://github.com/leanprover-community/mathlib-tools/).
+
+### lint
+
+A *linter* is a small program that looks for hard-to-spot mistakes in code.
+For [mathlib](#mathlib), we use [style linters](#style-linter)* and [code linters](#code-linter).
```

Comment:
I don't see a code linter entry, but I thought there was one before? 

Is the * after the style linter link Markdown for something?